x-pack/plugins/maps/server/saved_objects/map.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { SavedObjectsType } from 'src/core/server';
+import { APP_ICON, createMapPath } from '../../common/constants';
+// @ts-ignore
+import { migrations } from './migrations';
+
+export const mapSavedObjects: SavedObjectsType = {
+  name: 'map',
+  hidden: false,
+  namespaceType: 'single',
+  mappings: {
+    properties: {
+      description: { type: 'text' },
+      title: { type: 'text' },
+      version: { type: 'integer' },
+      bounds: { type: 'geo_shape' },
+      mapStateJSON: { type: 'text' },
+      layerListJSON: { type: 'text' },
+      uiStateJSON: { type: 'text' },
+    },
+  },
+  management: {
+    icon: APP_ICON,
+    defaultSearchField: 'title',
+    importableAndExportable: true,
+    getTitle(obj) {
+      return obj.attributes.title;
+    },
+    getInAppUrl(obj) {
+      return {
+        path: createMapPath(obj.id),
+        uiCapabilitiesPath: 'maps.show',
+      };
+    },
+  },
+  migrations: migrations.map,
+};
```

Comment:
I've actually also done this in [a draft PR in-progress](https://github.com/elastic/kibana/pull/65079). Is this work necessary for you to move forward?

Reply:
@aaronjcaldwell Any ideas when this can be merged? We can wait for a week.

Reply:
Actually, scratch that. I see you mentioned this is blocking https://github.com/elastic/kibana/issues/65461. Let's go with what you have done which is more fleshed out anyway. I'll work out the merge on the Maps end